cmd/publish_test.go

```diff
@@ -130,7 +133,7 @@ func TestCLI_Publish_Wait_PID_And_Cmd(t *testing.T) {
 	require.Equal(t, `command failed: does-not-exist-no-really "really though", error: exec: "does-not-exist-no-really": executable file not found in $PATH`, err.Error())
 
 	// Tests with NTFY_TOPIC set ////
-	require.Nil(t, os.Setenv("NTFY_TOPIC", topic))
```

Comment:
The tests after this one were failing because they kept connecting to the wrong ntfy server. It took me hours to figure out that this was the reason; the environment variable was persisting after this test and was affecting the remaining tests in the file.

Reply:
I didn't know you can do that `t.Setenv`. Wow. Sorry about that.